src/main/java/com/google/sps/exceptions/PlacesException.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.exceptions;
+
+public class PlacesException extends Exception {
```

Comment:
nit: Javadoc on the class and constructors

Reply:
Also wondering under what circumstances a PlacesException would be thrown - but I think that would/will be explained by the javadoc so feel free not to reply to this comment

Reply:
Good call on the javadoc comment. This exception is just to keep things consistent. For example, there's a DirectionsException for all things wrong with the Directions API. Naturally, I think it makes sense to have something identical for the Places API. Thanks!

Reply:
`RESOLVED`